install/0000_90_machine-api-operator_04_alertrules.yaml

```diff
@@ -52,3 +52,13 @@ spec:
             severity: critical
           annotations:
             message: "machine api operator metrics collection is failing. For more details:  oc logs <machine-api-operator-pod-name> -n openshift-machine-api"
+    - name: machine-health-check-unterminated-short-circuit
+      rules:
+        - alert: MachineHealthCheckUnterminatedShortCircuit
+          expr: |
+            mapi_machinehealthcheck_short_circuit == 1
+          for: 30m
+          labels:
+            severity: warning
+          annotation:
+            message: "machine health check {{ $labels.name }} has been short circuited for more than 30 minutes"
```

Comment:
If im not familer with MHC and I see this alert for the first time, how do I know what short circuited is?

Reply:
what if i change this to
```
machine health check {{ $labels.name }} has been disabled by short circuit for more than 30 minutes
```
would that make it clearer?
we do have a section in the official docs about short circuit, i'm just not sure how much to add in this alert message.

Reply:
It does make it clearer. Thanks :) 
Good point on docs, we should not pack these `messages` with too much info. 
https://docs.openshift.com/container-platform/4.7/scalability_and_performance/recommended-cluster-scaling-practices.html#machine-health-checks-short-circuiting_cluster-scaling